src/main/java/com/nokia/uwr/assigntmentsalgorithm/RandomAssignmentsAlgorithmImpl.java

```diff
@@ -0,0 +1,33 @@
+package com.nokia.uwr.assigntmentsalgorithm;
+
+import com.nokia.uwr.model.Measurements;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+@Component
+@Order(value = 2)
+public class RandomAssignmentsAlgorithmImpl implements AssignmentsAlgorithm {
+
+    private static final Logger LOGGER = LogManager.getLogger(RandomAssignmentsAlgorithmImpl.class);
+
+    @Override
+    public String findBTS(Measurements measurements) {
+        LOGGER.info("Find random BTS to connect to from UE: " + measurements.name());
+
+        Random random = new Random();
+        List<String> allBts = new ArrayList<>(measurements.signals().keySet());
```

Comment:
I think it would be enough: List<String> allBts = measurements.signals().keySet();

Reply:
But measurements.signals().keySet() returns set. And I want to have a list.